src/EnergyPlus/OutputProcessor.cc

```diff
@@ -2490,19 +2453,18 @@ namespace OutputProcessor {
 
         if (sql) {
             sql->createSQLiteReportDictionaryRecord(
-                reportID, storeType, indexGroup, keyedValue, variableName, timeStepType, unitsString, freq, false, ScheduleName);
+                ReportID, storeType, indexGroup, key, name, timeStepType, unitsString, freq, false, schedString);
         }
 
         // add to ResultsFramework for output variable list, need to check RVI/MVI later
-        rf->addReportVariable(keyedValue, variableName, unitsString, freq);
+        rf->addReportVariable(key, name, unitsString, freq);
 
-    } // WriteReportVariableDictionaryItem()
+    } // OutVar::WriteReportDictionaryItem()
 
     void WriteMeterDictionaryItem(EnergyPlusData &state,
```

Comment:
Wanted to make this into a method too, but this case is a little more complicated so left it alone for now.